src/index.js

```diff
@@ -62,11 +63,11 @@ module.exports = ({ tableName, ddbClient }) => ({
         //console.log('returned from db', items)
         return count > 0
           ? { aggregate: { ...(items.slice(-1)[0]), aggregateId }, events: items.slice(0, -1) }
-          : { aggregate: { version: 0, aggregateTypeName, aggregateId }, events: [] }
+          : { aggregate: { version: ulid(), aggregateTypeName, aggregateId }, events: [] }
```

Comment:
I think it would be better to preserve the aggregate Version but use a monotonically increasing ulid in the Sort key instead of the version.
So let's preserve `version` as a monotonically increasing number, and introduce `versionUlid` in the aggregate record.